public/static/sdk/jsmr-0.1.js

```diff
@@ -145,22 +196,113 @@ var JsMr = Class.extend({
         );
     },
 
+    getData: function(task) {
+
+
+        var output = "";
+
+        var context = {
+            write: function (key, value) {
+                output = output.concat(key, ":", value, "\t");
```

Comment:
since we're reading data line by line, let's be consistent and use `\n` to separate the outputs.